resources/assets/js/annotations/components/annotationCanvas/polygonBrushInteraction.vue

```diff
@@ -100,33 +100,44 @@ export default {
             }
         },
     },
-    created() {
-        if (this.canAdd) {
-            Keyboard.on('e', this.togglePolygonBrush, 0, this.listenerSet);
-        }
-
-        if (this.canModify) {
-            Keyboard.on('r', this.togglePolygonEraser, 0, this.listenerSet);
-            Keyboard.on('t', this.togglePolygonFill, 0, this.listenerSet);
-        }
-
-        if (this.canAdd || this.canModify) {
-            this.$watch('interactionMode', this.toggleCurrentInteraction);
+    watch: {
+        interactionMode(mode) {
+            if (this.canAdd || this.canModify) {
+                this.toggleCurrentInteraction(mode);
+            }
```

Comment:
No need for `if` here because it's already handled in `toggleCurrentInteraction()`.

```suggestion
            this.toggleCurrentInteraction(mode);
```